webots_ros2_tutorials/protos/RobotSense.proto

```diff
@@ -0,0 +1,210 @@
+#VRML_SIM R2020b utf8
+# license: Apache License 2.0
+# license url: http://www.apache.org/licenses/LICENSE-2.0
+# tags: static
+# More info here: https://www.youtube.com/c/soft-illusion
+# Robotsense
+
+PROTO RobotSense [
+    field SFVec3f    translation  -0.143941 0.0198605 0.202666
+    field SFRotation rotation     1.0911549263414444e-05 0.9999999999355068 3.1502938785775475e-06 2.349517532397045
+    field SFString   controller   "<extern>"
+  ]
+  {
+    Robot {
+  translation IS translation
+  rotation IS rotation
+  controller IS controller
+  children [
+    DistanceSensor {
+      type "infra-red"
+      translation -0.03 -0.02 0.1
+      rotation 0 0 -1 1.5708
+      children [
+        Shape {
+          appearance PBRAppearance {
+            baseColor 0.988235 0.913725 0.309804
+          }
+          geometry Box {
+            size 0.01 0.01 0.01
+          }
+        }
+      ]
+      name "ls_right"
```

Comment:
The name looks strange here, what does mean "ls" light sensor? That seems wrong, because it's a distance sensor. In general, we try to avoid abbreviations to make things clearer.

Reply:
Done